tests/sqllogictests/suites/query/02_function/02_0069_experimental_expr.test

```diff
@@ -15,7 +15,7 @@ OPENAI IS NOTHING WITHOUT ITS PEOPLE
 query IIII
 with t(f) as (select '11|open|22|ai|33|is nothing without sam'.split('|')
         .array_filter(x -> try_cast(x as int) is not null ).array_apply(x -> x::int ).array_apply(x -> x::int64 ))
-	select f.array_reduce('sum'), f.array_reduce('max'), f.array_reduce('min')  from t;
```

Comment:
`array_reduce` is probably already used by some users, so it's not a good idea to rename it.